lib/components/hero/index.tsx

```diff
@@ -0,0 +1,40 @@
+import { Image, Heading, Button, Flex } from '@chakra-ui/react';
+import Link from 'next/link';
+import React from 'react';
+
+interface HeroProps {
+  imgSrc: string;
+  text: string;
+}
+
+const Hero: React.FC<HeroProps> = ({ imgSrc, text }) => {
+  return (
+    <div data-cy="learn-more-section" style={{ position: 'relative' }}>
+      <Image width="100%" height="100vh" objectFit="cover" alt="Learn More Image" src={imgSrc} />
+      <Flex
+        gap="1rem"
```

Comment:
Have you looked at this? https://www.w3schools.com/howto/howto_css_image_center.asp
Try not to use rem either :) pixels are preferred